types/geometric/index.d.ts

```diff
@@ -0,0 +1,198 @@
+// Type definitions for geometric 2.2
+// Project: https://github.com/HarryStevens/geometric#readme
+// Definitions by: Linda Paiste <https://github.com/DefinitelyTyped>
```

Comment:
```suggestion
// Definitions by: Linda Paiste <https://github.com/lindapaiste>
```
unless you want to stay off-the grid as the owner

Reply:
I'm fine attaching myself to it.  That was just an oversight.  Thanks for catching it!